src/openforms/js/components/form/fieldset.js

```diff
@@ -21,7 +21,13 @@ const FIELDSET_BASIC = {
             key: 'hidden',
             label: 'Hidden',
             tooltip: 'A hidden field is still a part of the form, but is hidden from view.'
-        }
+        },
+        {
+            type: 'checkbox',
+            key: 'clearOnHide',
+            label: 'Clear on hide',
+            tooltip: 'Remove the value of this field from the submission if it is hidden',
+        },
```

Comment:
a fieldset doesn't have a value right, but then I'd assume the entire nested object with values is cleared? Can you confirm that?

Reply:
If you have a hidden fieldset then all the values of the children don't appear in the submission